package.json

```diff
@@ -98,6 +98,7 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
     "mocha": "^2.2.5",
+    "nodemon": "^1.9.1",
```

Comment:
@oliviertassinari Yep, probably to the `contributing.md` guide. 

However, even though it states that in the docs. It **is totally fine** to add it to dev dependencies. In fact, `nodemon` just removed the `preferGlobal: true` option last month: https://github.com/remy/nodemon/issues/772

The reasoning behind preferring global would not affect us as we are using `npm scripts` to call it.

I added it the way I did out of habit really -- I use the nodemon CLI directly **a lot** so have it globally installed.